File: two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> results= vector<int>(2);
        int l= nums.size();
        for (int i=0; i<l; i++)
            for (int j=i+1; j<l; j++)
                if (nums[i]+ nums[j] == target){
                    results[0] = i+1;
                    results[1] = j+1;
                    return results;
                }
    }
};

File: python/array and string/two-sum.py
#https://leetcode.com/problems/two-sum/

class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        d= {}
        for i, x in enumerate(nums):
            y = target - x 
            if y in d:
                return [d[y], i]
            else:
                d[x]= i
                
